### ISSUE TITLE
Manual activation of company backend

### ISSUE THREAD

**Issue by dsnipe** [tonini/alchemist.el#101]
It's great plugin and one of the reason why I switched to  Emacs :) Thanks for that!
But one thing is annoying me – company-mode shows only alchemist backend (I think it's related to #66). So my suggestion is not activate backend automatically, but give necessary code in documentation. 
At least any suggestions how to disable backend?
I tried add hook where remove that backend, but without succeed.

**Comment by tonini** [comment]
Hey @dsnipe 

Ok I updated on the current master branch `alchemist-company` that the alchemist backend will be added to the `company-backends` variable. https://github.com/tonini/alchemist.el/blob/master/alchemist-company.el#L77

This is the way all company-modes doing it. And it works without any problems. Company-mode uses the next company backends from the list when prefix is nil. In alchemist-company we handle that exactly that way.

I tested it by myself with different major modes,  like switch from ruby to emacs lisp and to elixir etc and it worked all the time.

I will have a look into more company-modes and also talk with some emacs lisp experts I know.

Thanks for your patiences and I hope this issue won't prevent you from having fun with hacking Elixir with Emacs and Alchemist.

Cheers :heart: :yellow_heart: :blue_heart:
